extras/library_generation/library_generation.sh

```diff
@@ -249,6 +250,20 @@ if [[ " ${PLATFORMS[@]} " =~ " esp32 " ]]; then
     cp -R firmware/build/libmicroros.a /project/src/esp32/libmicroros.a
 fi
 
+######## Build for ESP32 with toolchain v5.2.0  ########
+if [[ " ${PLATFORMS[@]} " =~ " esp32_5_2_0 " ]]; then
+    rm -rf firmware/build
+
+    export TOOLCHAIN_PREFIX=/uros_ws/xtensa-esp32-elf-linux64-1.22/bin/xtensa-esp32-elf-
+    ros2 run micro_ros_setup build_firmware.sh /project/extras/library_generation/esp32_toolchain.cmake /project/extras/library_generation/colcon.meta
+
+    find firmware/build/include/ -name "*.c"  -delete
+    cp -R firmware/build/include/* /project/src/
+
+    mkdir -p /project/src/esp32_5_2_0
```

Comment:
Will Arduino IDE look in this folder for the library?

Reply:
Only with additional flags:
```
-L <path_to_microros>/src/esp32_5_2_0/
-l microros